docs/randomblind.md

```diff
@@ -0,0 +1,139 @@
+:----------------------------:|:----------------------------:
+![Permission](assets/rb_permission.jpg) | ![Card](assets/rb_card.jpg)
+
+## Cryptography
+In this section we will explain in detail how the issuer and the user are able
+to perform such an issuance session. Randomblindness of attributes only concerns
+the issuance protocol. The disclosure and signature protocols of involving radomblind attributes
+is in every aspect similar to disclosing or signing using a normal attribute.
+
+We can assume that both parties have 
+a scheme in which the indices of the randomblind attributes within the
+credentialtype coincide. If this is not the case, either party cancels the
+session. In the following example, we only have one randomblind attribute, but
+the protocol can be extended straightforwardly to issue multiple randomblind
+attributes.
+
+TODO: link to intro on CL-signatures.
+
+Variables $R, S, Z$ are public parameters.
+
+The issuance goes as follows:
+- The user starts by a sampling $v'$ and a random 255-bit integer $m_{r}'$ at
+  the index $r$ of the randomblind attribute.
+  This so-called *share* of the attribute remains secret, similar to
+  how the user's secret key remains secret during issuance. 
+  The user computes the commitment $U = S^{v'} R_0^{m_0} R_r^{m_r} \mod n$.
+  Note that $m_0$ is always the user's secret key. This commitment is sent to
+  the issuer along with a [zero-knowledge proof](https://irma.app/docs/zkp/)
+  of not just $v'$ and $m_0$, but also over this share $m_{r}'$.
+
+- The issuer samples a random prime $e$. The issuer also samples $v''$ and $m_{r}''$ at random.
+  Since attribute values ($m$'s) are at most 256 bits, $m_{r}''$ is also 255 bits (to not overflow).
+  Next, the issuer computes 
+
+  $$
+    A = \Bigg(\frac{Z}{U S^{v''} R_{1}^{m_1} \dots \ R_{r}^{m_r''} \dots \ R_{n}^{m_n}}\Bigg)^{1/e}
+  $$
+
+  For every attribute that is tagged randomblind, the issuer includes its share
+  of the attribute in the exponent. In this case, only for index $r$.  For
+  regular attributes, the exponent is simply the attribute value that the
+  issuer wants to pass. The issuer sends $(A, e, v'', m_{r}'')$ back to the
+  user, again, along with a zero-knowledge proof of $A$.
+
+- Finally, the user is able to construct a valid CL-signature for the credential
+  by summing up the shares:
+  
+  $$
+  v = v' + v'' \\
+  m_r = m_r' + m_r''
+  $$
+
+  The CL signature becomes $(A, e, v)$. It follows that $Z = A^e S^v
+  \prod_{i=0}^{l} R_{i}^{m_i}$ holds for the attributes $m$ in the resulting
+  credential, meaning that the credential is valid.  We  also note that the
+  issuer does not have enough information to fully uncover $m_r$, which ends up
+  as the attribute value in the resulting credential. The user is now free to
+  use this attribute, knowing the issuer cannot trace it back to him/her.
+  Note that after revealing this attribute once, this guarantuee no longer holds.
```

Comment:
This - the single use of these attributes - is such an incredible point that it should be mentioned close to the top of this page, and not just here (at the very bottom).

In addition, I think it would be nice if there would be a paragraph explaining a little more fully the election use case.